tests/contrib/falcon/test_suite.py

```diff
@@ -192,3 +193,21 @@ def test_200_ot(self):
         eq_(dd_span.resource, 'GET tests.contrib.falcon.app.resources.Resource200')
         eq_(dd_span.get_tag(httpx.STATUS_CODE), '200')
         eq_(dd_span.get_tag(httpx.URL), 'http://falconframework.org/200')
+
+    def test_http_header_tracing(self):
+        self.simulate_get('/200', headers={'my-header': 'my_value'})
+        traces = self.tracer.writer.pop_traces()
+        eq_(len(traces), 1)
+        eq_(len(traces[0]), 1)
+        span = traces[0][0]
+        eq_(span.get_tag('http.request.headers.my_header'), None)
+        eq_(span.get_tag('http.response.headers.my_response_header'), None)
+
+        config.falcon.http.trace_headers(['my-header', 'my-response-header'])
```

Comment:
We'll want to reset this value after this test has run.

There is a helper available on `0.16-dev` for this: https://github.com/DataDog/dd-trace-py/blob/0.16-dev/tests/util.py#L66-L84